README.md

```diff
@@ -75,95 +66,135 @@ yarn add @fpjs-incubator/botd-agent
 ```
 
 ```js
-import Botd from '@fpjs-incubator/botd-agent';
+import { load } from '@fpjs-incubator/botd-agent'
 
 // Initialize an agent at application startup.
-const botdPromise = Botd.load({ publicKey: '<your-public-key>' });
-
-(async () => {
-  // Get the bot detection result when you need it.
-  // Result will contain the `requestId` property, that you can securely verify on the server.
-  const botd = await botdPromise;
-  const result = await botd.detect();
-  console.log(result);
-})();
+load()
+    .then((botd) => botd.detect())
+    .then((result) => console.log(result))
+    .catch((error) => console.error(error))
 ```
-[Run this code](https://stackblitz.com/edit/botd-npm?devtoolsheight=100&file=index.js)
 
-### 📕 [Full documentation](docs/api.md)
+[Run this code](https://stackblitz.com/edit/botd-cdn?devtoolsheight=100&file=index.html)
+
+📕 [Full documentation](docs/api.md)
 
-## Authentication
+## 🤖 Use the free Pro version to get sophisticated bot detection
 
-You need a pair of keys to use BotD:
-- `publicKey` for making bot detection requests from browser. This key can be used publicly on your websites.
-- `secretKey` for verifying bot detection requests on the server. This key must be kept secret.
+[Fingerprint Pro](https://fingerprint.com/products/bot-detection/) is a professional bot detection service that processes all information server-side and transmits it securely to your servers using server-to-server APIs.
+Pro combines vast amounts of auxiliary data that bots leak (cursor movements, network overrides, browser changes and more) to be able to reliably deduplicate real users from automated software, resulting in the detection of popular automation tools, their derivatives and plugins.
 
-_Please use [the following form](https://fingerprint.com/botd#generateKeySection) on our [BotD product page](https://fingerprint.com/products/bot-detection/) to generate your keys._
-<br/>
+**Pro plans start at $0/month - no credit card required.**
 
-### **Request Limit**
-_The free keys are limited to 3M API calls per month and 10 calls per second while in beta._
+<br />
 
-## Supported detection scenarios
+<p align="center">
+  <a href="https://fingerprint.com/products/bot-detection/">
+    <img src="resources/pro_botd_screenshot.png" alt="Pro BotD screenshot" width="512px" />
+  </a>
+</p>
 
-### **Automation Tools & Frameworks**
+<br />
+<br />
+
+Full product comparison:
```

Comment:
I think the comparison of products makes sense to put in a separate document, and here to concentrate on the description of the BotD OSS, what do you think?

Reply:
I would leave it here, it gives us higher conversion rate

Reply:
Ok, I'm sticking with comparison.